maven-model/src/main/java/org/apache/maven/model/io/xpp3/MavenXpp3ReaderEx.java

```diff
@@ -112,13 +119,13 @@ public Model read(InputStream in, InputSource source) throws IOException, XmlPul
      *
      * @param parser a parser object.
      * @param strict a strict object.
-     * @throws IOException IOException if any.
-     * @throws XmlPullParserException XmlPullParserException if
-     * any.
      * @return Model
+     * @throws IOException            IOException if any.
```

Comment:
no period, and in general "if any" is not sufficient description. Something like "if an I/O error occurs while reading from the underlying source"